src/lib/protect/apply-patch.js

```diff
@@ -5,7 +5,7 @@ const exec = require('child_process').exec;
 const path = require('path');
 const fs = require('fs');
 const semver = require('semver');
-const errorAnalytics = require('../analytics').postAnalytics;
+const analytics = require('../analytics');
```

Comment:
Could we use modules?

```suggestion
import { addDataAndSend }  from '../analytics';
```

Reply:
👍